{
    "dataType": "CVE_RECORD",
    "cveMetadata": {
        "state": "PUBLISHED",
        "cveId": "CVE-2025-57833",
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "dateUpdated": "2025-09-03T20:15:16.887Z",
        "dateReserved": "2025-08-20T00:00:00.000Z",
        "datePublished": "2025-09-03T00:00:00.000Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unknown",
                    "product": "Django",
                    "vendor": "djangoproject",
                    "versions": [
                        {
                            "lessThan": "4.2.24",
                            "status": "affected",
                            "version": "4.2",
                            "versionType": "custom"
                        },
                        {
                            "lessThan": "5.1.12",
                            "status": "affected",
                            "version": "5.1",
                            "versionType": "custom"
                        },
                        {
                            "lessThan": "5.2.6",
                            "status": "affected",
                            "version": "5.2",
                            "versionType": "custom"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "value": "An issue was discovered in Django 4.2 before 4.2.24, 5.1 before 5.1.12, and 5.2 before 5.2.6. FilteredRelation is subject to SQL injection in column aliases, using a suitably crafted dictionary, with dictionary expansion, as the **kwargs passed QuerySet.annotate() or QuerySet.alias()."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-89",
                            "description": "CWE-89 Improper Neutralization of Special Elements used in an SQL Command ('SQL Injection')",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre",
                "dateUpdated": "2025-09-03T20:15:16.887Z"
            },
            "references": [
                {
                    "url": "https://docs.djangoproject.com/en/dev/releases/security/"
                },
                {
                    "url": "https://groups.google.com/g/django-announce"
                },
                {
                    "url": "https://www.djangoproject.com/weblog/2025/sep/03/security-releases/"
                }
            ],
            "x_generator": {
                "engine": "enrichogram 0.0.1"
            },
            "metrics": [
                {
                    "cvssV3_1": {
                        "version": "3.1",
                        "baseScore": 7.1,
                        "baseSeverity": "HIGH",
                        "vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:C/C:H/I:L/A:N"
                    }
                }
            ],
            "cpeApplicability": [
                {
                    "nodes": [
                        {
                            "operator": "OR",
                            "negate": false,
                            "cpeMatch": [
                                {
                                    "vulnerable": true,
                                    "criteria": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                    "versionStartIncluding": "4.2",
                                    "versionEndExcluding": "4.2.24"
                                },
                                {
                                    "vulnerable": true,
                                    "criteria": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                    "versionStartIncluding": "5.1",
                                    "versionEndExcluding": "5.1.12"
                                },
                                {
                                    "vulnerable": true,
                                    "criteria": "cpe:2.3:a:djangoproject:django:*:*:*:*:*:*:*:*",
                                    "versionStartIncluding": "5.2",
                                    "versionEndExcluding": "5.2.6"
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    },
    "dataVersion": "5.1"
}